### DISCUSSION TITLE
404 Error when accessing all my Gists

### DISCUSSION THREAD

**Discussion by Spliitsub** [community/community#132366] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello GitHub Support,

I am experiencing a 404 error when trying to access any of my Gists. I have double-checked multiple URLs and they all appear to be correct. I have also tried clearing my browser cache and accessing the Gists from a different device, but the issue persists.

For example, this is the URL for one of my Gists: https://gist.github.com/Spliitsub/64539d273a7fc5be4590e279958e1bab

Could you please investigate this issue and help me resolve it? I am concerned that there may be a broader problem with my account or with GitHub's Gist service.

Thank you for your time and assistance.

**Comment by L1gor4ik** [comment]
Hello! How you solve this problem? I have exactly same problem as you (
